src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api/UnitTestingProjectExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal static class UnitTestingProjectExtensions
+    {
+        public static string GetDebugName(this ProjectId projectId)
+            => projectId.DebugName;
+
+        public static Task<bool> UnitTesting_HasSuccessfullyLoadedAsync(this Project project, CancellationToken cancellationToken = default)
+            => project.HasSuccessfullyLoadedAsync(cancellationToken);
```

Comment:
📝 Call this directly instead of as an extension method to avoid stack overflow

Reply:
Looks like this is a common method so I can directly call it.